### ISSUE TITLE
Not saving patches when non latin charset is involved

### ISSUE THREAD

**Issue by dilom** [VCVRack/Rack#1887]
I was recently using VCV on a Win 10 system and noticed that my patches weren't saving at all. No warning or error was given at all to indicate that there might be a problem, however the patch files are not present at their saved location or anywhere else for that matter. To add to the fun the "saved" patch files were in fact listed under the Recently used files in Win 10 with the correct path. However no such files exist at that path location. 

After some confusion I decided to check if somebody here had reported anything similar and I think I've got some idea what the issue is. One of the closed tickets was mentioning a problem with saving (JSON seemed to be involved) and non UTF-8 characters. Then I noticed that the account/username in the particular Win 10 system that had the issue was NOT using Latin characters, so I tried to save the same patch from the same account but to a path name that did not include the said account name in its path. Everything saved as it should and the file was at its location. After that I copied the same saved patch file back to the location that included the non latin charset in its path. The file copied correctly, however when double clicked to open it, a totally empty rack loaded with 0 modules of anything (no error or other message was given). I then copied the same patch file back to its previous location, double clicked it and everything loaded correctly in VCV.  

Bottom line seems to be that VCV needs better handling and support of non latin charsets.

Cheers

**Comment by AndrewBelt** [comment]
This issue is a subset of https://github.com/VCVRack/Rack/issues/436 which has been resolved in Rack v2.
